spec/fibonacci.js

```diff
@@ -0,0 +1,17 @@
+// import chai, { expect} from 'chai'
+// import chaiChange from 'chai-change'
+// import Fibonacci from '../src/fibonacci'
+
+// chai.use(chaiChange)
+
+// describe('Fibonacci', () => {
```

Comment:
Did you comment out to not return tests that don't pass? You can achieve testing a specific file/function by using the describe.only method. Just remember to remove the .only when you move on to the next one.